### ISSUE TITLE
Simulation: Refactor "Results" Into a "FlightData" function to be able to store sim results

### ISSUE THREAD

**Issue by TheGreatFez** [TSoGiants/TSoG_X1#3]
Dependent on #23 

Rename our current "Results" data structure, for saving the simulation, to "FlightData".

Inputs should be:
- Previous Flight Data structure
- Current State Vector

Outputs should be:
- New Flight Data Structure

**Comment by Nufflee** [comment]
I'm not exactly sure what you mean here. I guess you want there to be a function that will be used to "evolve" the state with inputs and outputs that you mentioned but what does that have to do with renaming `Results` to `FlightData`?

**Comment by TheGreatFez** [comment]
No, not evolve the state just separate out the recording of the simulation (the information we care about at least) to its own function. This way as we add data to store over time, we don't need to add them into the main sim function.

**Comment by adiojha629** [comment]
Given that this uses StateVector, lets wait for issue #23 to get done and then complete this

**Comment by TheGreatFez** [comment]
This issue is no longer relevant to our current Simulation Tools architecture. Closing.
